securedrop/tests/functional/app_navigators/journalist_app_nav.py

```diff
@@ -91,7 +90,7 @@ def journalist_checks_messages(self) -> None:
         collections_count = self.count_sources_on_index_page()
         assert collections_count == 1
 
-        if not self.accept_languages:
+        if self.accept_languages in [None, "en_US"]:
```

Comment:
DRY nit: How would you feel about adding a utility function like:

```python
def test_en(lang: str):
    return lang in [None, "en_US"]
```

And then all of these checks would reduce to:

```suggestion
        if test_en(self.accept_languages):
```

Reply:
I was actually thinking of going one step further and having like a `localized_string_is_found()` dealy, which could in the future actually look up the translated string, but thought it was overkill.

Reply:
For now I'd probably leave it as is, so the check is explicit.